src/nagendra/BankingOpration.java

```diff
@@ -0,0 +1,161 @@
+package nagendra;
+
+public class BankingOpration {
+	int iciciBalance = 1000;
+	int hdfcBalance = 1000;
+	int bobBalance = 1000;
+	int sbiBalance = 1000;
+	int iciciCredit;
+	int hdfcCredit;
+	int bobCredit;
+	int sbiCredit;
+	int iciciDebit;
+	int hdfcDebit;
+	int bobDebit;
+	int sbiDebit;
+
+	void bankOperation(String bank, String operation, int amount) {
+		switch (bank) {
+		case "ICICI":
+			if (operation.equalsIgnoreCase("Credit"))
+				creditOperation(bank, amount);
+			else if (operation.equalsIgnoreCase("Debit"))
+				debitOperation(bank, amount);
+			break;
+		case "HDFC":
+			if (operation.equalsIgnoreCase("Credit"))
+				creditOperation(bank, amount);
+			else if (operation.equalsIgnoreCase("Debit"))
+				debitOperation(bank, amount);
+			break;
+		case "BOB":
+			if (operation.equalsIgnoreCase("Credit"))
+				creditOperation(bank, amount);
+			else if (operation.equalsIgnoreCase("Debit"))
+				debitOperation(bank, amount);
+			break;
+		case "SBI":
+			if (operation.equalsIgnoreCase("Credit"))
+				creditOperation(bank, amount);
+			else if (operation.equalsIgnoreCase("Debit"))
+				debitOperation(bank, amount);
+			break;
+		}
+	}
+	void creditOperation(String bank, int amount) {
+		switch (bank) {
+		case "ICICI":
+			if (amount > iciciBalance)
+				iciciBalance += amount;
+			iciciCredit++;
+			break;
+		case "HDFC":
+			hdfcBalance += amount;
+			hdfcCredit++;
+			break;
+		case "BOB":
+			bobBalance += amount;
+			bobCredit++;
+			break;
+		case "SBI":
+			sbiBalance += amount;
+			sbiCredit++;
+			break;
+		}
+	}
+
+	void debitOperation(String bank, int amount) {
+		switch (bank) {
+		case "ICICI":
+			if (amount > iciciBalance) {
+				System.out
+						.println("The amount " + amount + " is not available in Your "+bank+" Account total Available Balance is " + iciciBalance + " ");
+			} else
+				iciciBalance -= amount;
+			iciciDebit++;
```

Comment:
here this iciciDebit should be in bracket with else, bcoz if u see if amount > iciciBalace case still we are increasing the iciciDebit variable which looks wrong, think, you wan to increase counter when you debit from account right